src/helpers/wrapPetitionLinks.js

```diff
@@ -0,0 +1,8 @@
+export default (links) => {
+  return links && links
+    .filter(link => link)
+    .map(link => ({
+      url: link.data ? link.data.url : link.url,
+      og: link.data ? link.data.og : link.og
+    }));
```

Comment:
Gotta translate the data because it’s the only API POST endpoint that doesn’t want the data wrapped in a data object…